apps/imageHotspotCreator/src/components/Home/home.tsx

```diff
@@ -24,14 +24,42 @@ const Usi = ({ sdk }: any) => {
   const [imageAssets, setImageAssets] = useState<[]>()
  
 
+  function checkImageURL(url:any, callback:any) {
+    var img = new Image();
+  
+    img.onload = function() {
+      callback(true);
+    };
+  
+    img.onerror = function() {
+      callback(false);
+    };
+  
+    img.src = url;
+  }
+
   //This UseEffect is used for the first time if there is already values for that entry
   useEffect(() => {
-    if (sdk?.entry?.fields?.imageUrl?.getValue()) {
-      const url = sdk.entry.fields.imageUrl.getValue()
-      setImageUrl(url)
-      setImageName(sdk.entry.fields.title.getValue())
-      setSelectedImage(sdk.entry.fields.title.getValue())
-      setImageStatus(true)
+    const url = sdk?.entry?.fields?.imageUrl?.getValue()
+    if (url) {
+      checkImageURL(url, function(isValid:boolean) {
+        if (isValid) {
+          console.log('The image URL is valid.');
+          setImageUrl(url)
+          setImageName(sdk.entry.fields.title.getValue())
+          setSelectedImage(sdk.entry.fields.title.getValue())
+          setImageStatus(true)
+        } else {
+          console.log('The image URL is invalid.');
```

Comment:
Same question: Is this console log still needed here?

Reply:
Not Needed